shim.h

```diff
@@ -60,6 +60,7 @@ extern long X_SSL_CTX_sess_get_cache_size(SSL_CTX* ctx);
 extern int X_SSL_CTX_set1_curves(SSL_CTX *ctx, int *clist, int clistlen);
 extern long X_SSL_CTX_add_extra_chain_cert(SSL_CTX* ctx, X509 *cert);
 extern long X_SSL_CTX_set_tlsext_servername_callback(SSL_CTX* ctx, int (*cb)(SSL *con, int *ad, void *args));
+extern int X_SSL_CTX_set1_groups_list(SSL_CTX* ctx, void *s);
```

Comment:
[manpage](https://docs.openssl.org/master/man3/SSL_CTX_set1_curves/) says this is a `char *`, not a `void *`
> `int SSL_CTX_set1_groups_list(SSL_CTX *ctx, char *list);`

Reply:
Yep, my mistake. Pushed a fix.